lib/commands/emulate.ts

```diff
@@ -1,17 +1,36 @@
 export class EmulateCommandBase {
-	constructor(private $platformService: IPlatformService) { }
+	constructor(private $options: IOptions,
+		private $projectData: IProjectData,
+		private $platformService: IPlatformService) {
+			this.$projectData.initializeProjectData();
+		}
 
 	public async executeCore(args: string[]): Promise<void> {
-		return this.$platformService.emulatePlatform(args[0]);
+		this.$options.emulator = true;
```

Comment:
As I have not removed the `$options` dependency from **all** services I cannot guarantee that the code will run the same if I remove this.